packages/@aws-cdk/aws-lambda/test/function.test.ts

```diff
@@ -1873,8 +1852,8 @@ describe('function', () => {
       });
 
       // THEN
-      const template1 = SynthUtils.synthesize(stack1).template;
-      const template2 = SynthUtils.synthesize(stack2).template;
+      const template1 = Template.fromStack(stack1).toJSON();
```

Comment:
Extremely nitpicky and also too late I suppose, but that's not what "JSON" means.

This is not JSON:

```
{ banana: "yellow" }
```

**This** is JSON:

```
"{ \"banana\": \"yellow\" }"
```

The same way that `é` is not UTF-8, but `C3 A9` is UTF-8. 

JSON is not the *data*, it's an *encoding*.

So the output of `toJSON()` needs to be a `string`, not an `object`.